ref/net46/Microsoft.Build.Framework/Microsoft.Build.Framework.cs

```diff
@@ -48,9 +48,9 @@ public partial class BuildEventContext
         public BuildEventContext(int nodeId, int targetId, int projectContextId, int taskId) { }
         public BuildEventContext(int nodeId, int projectInstanceId, int projectContextId, int targetId, int taskId) { }
         public BuildEventContext(int submissionId, int nodeId, int projectInstanceId, int projectContextId, int targetId, int taskId) { }
-        public BuildEventContext(int submissionId, int nodeId, int evaluationID, int projectInstanceId, int projectContextId, int targetId, int taskId) { }
+        public BuildEventContext(int submissionId, int nodeId, int evaluationId, int projectInstanceId, int projectContextId, int targetId, int taskId) { }
         public long BuildRequestId { get { throw null; } }
-        public int EvaluationID { get { throw null; } }
+        public int EvaluationId { get { throw null; } }
```

Comment:
We don't have an insertion yet. CPS has the nuget packages, though I don't know if they started coding it yet.